src/commands/importDocuments.ts

```diff
@@ -57,14 +57,18 @@ export async function importDocuments(uris: vscode.Uri[] | undefined, collection
 }
 
 async function askForDocuments(): Promise<vscode.Uri[]> {
-    return await ext.ui.showOpenDialog({
+    let openDialogOptions: vscode.OpenDialogOptions = {
         canSelectMany: true,
         openLabel: "Import",
         filters: {
             "JSON": ["json"]
-        },
-        defaultUri: vscode.Uri.file(vscode.workspace.rootPath)
```

Comment:
'Course, you could've just done this:

  defaultUri: vscode.workspace.rootPath ? vscode.Uri.file(vscode.workspace.rootPath) : undefined